tests/components/test_score.py

```diff
@@ -0,0 +1,144 @@
+from uuid import uuid4
+
+from lenskit.pipeline import PipelineBuilder
+
+from poprox_concepts.domain import Article, CandidateSet, InterestProfile, RecommendationList
+from poprox_recommender.components.joiners.score import ScoreConfig, ScoreFusion
+
+# def test_trail():
+#     article=[]
+#     n_scorer=CandidateSet(articles=article)
+#     n_scorer2=CandidateSet(articles=article)
+#     fusion=ScoreFusion()
+#     output=fusion(candidates1=n_scorer, candidates2=n_scorer2)
+#     assert output is not None
+
```

Comment:
I think this test was useful to motivate building out the structure of the class, but we can remove it now:
```suggestion
```